src/network/networkprotocol.h

```diff
@@ -629,6 +633,11 @@ enum ToClientCommand
 		float step
 		float gain
 	*/
+	TOCLIENT_UPDATE_PLAYER_LIST = 0x56,
+	/*
+	 	 enum type;
```

Comment:
`enum` does not have a defined size
you need to make sure in the code to use `u32` (or whatever else) because this could break on other platforms